services/render.tsx

```diff
@@ -87,37 +102,39 @@ function EmbeddedMedia(node: any, isInline = false): ReactElement {
         figureStyles.width = imageWidth;
       }
       return (
-        <figure className={classNames(figureClasses)} style={figureStyles}>
-          <Image
-            src={`${url}${isSmall ? '?w=300' : isMedium ? '?w=600' : ''}`}
-            alt={node.title}
-            width={imageWidth}
-            height={imageHeight}
-            priority={true}
-          />
-          {node.caption && (
-            <figcaption className={classNames(captionClasses)}>
-              <em>
-                {node.sourceUrl ? (
-                  <Link href={node.sourceUrl}>
-                    <a
-                      aria-label={node.caption}
-                      className={classNames(
-                        'text-primary-dark font-extralight'
-                      )}
-                      target="_blank"
-                      rel="noreferrer"
-                    >
-                      {node.caption}
-                    </a>
-                  </Link>
-                ) : (
-                  <>{node.caption}</>
-                )}
-              </em>
-            </figcaption>
-          )}
-        </figure>
+        <span className={classNames(figureClasses)} style={figureStyles}>
```

Comment:
Do we need this span?

Reply:
remembered why I added the span, because those figureStyles in some cases have a flex items-center style, which messed up the rtl direction in the figcaption and centered it always and the way I worked around it was creating this wrapper with the center styles in it and not the figure tag.

Reply:
Ok cool. Works for me.